js/preset_definitions.js

```diff
@@ -1157,489 +1157,328 @@ presets.presets = [
-            {
+                value: 10
+	},
+	{ 
                 key: "fw_ff_pitch",
-                value: 56
-            },
-            {
+                value: 60
+	},
+	{ 
                 key: "fw_p_roll",
-                value: 7
-            },
-            {
+                value: 10
+	},
+	{ 
                 key: "fw_i_roll",
-                value: 12
-            },
-            {
+                value: 8
+	},
+	{ 
                 key: "fw_ff_roll",
-                value: 25
-            },
-            {
-                key: "rc_expo",
-                value: 30
-            },
-            {
-                key: "manual_rc_expo",
-                value: 30
-            },
-            {
+                value: 40
+	},
+	{ 
+                key: "fw_p_yaw",
+                value: 20
+	},
+	{ 
+                key: "fw_i_yaw",
+                value: 0
+	},
+	{ 
+                key: "fw_ff_yaw",
+                value: 100
+	},
+	{
                 key: "imu_acc_ignore_rate",
                 value: 10
-            }
-        ],
-        type: 'flyingwing'
+	},
+	{
+                key: "airmode_type",
+                value: "STICK_CENTER_ONCE"
+	},
+	{
+                key: "small_angle",
+                value: 180
+	},
+	{
+                key: "nav_fw_control_smoothness",
+                value: 2
+	},
+	{
+                key: "nav_rth_allow_landing",
+                value: "FS_ONLY"
+	},
+	{
+                key: "nav_rth_altitude",
+                value: 5000
+	},
+	{
+                key: "failsafe_mission",
+		value: "ON"
+	},
+	{
+                key: "nav_wp_radius",
+                value: 1500
+	},
+		],
     },
     {
-        name: "ZOHD Dart 250g",
-        description: "3x5x3 propeller, 1406 2600kV motor, 3S LiPo. 570mm wingspan, AUW potentially under 250g on 2S.<br /><br /><strong>Please set the Stabilised Roll weight to 80, and the Stabilised Pitch weight to 65.</strong>",
-        features: [
-            "Adjusted gyro filtering",
-            "Adjusted PIDs",
-            "Adjusted rates"
-        ],
+        name: "Airplane without tail",
+        description: "General setup for airplanes without tails: Flying Wing, Delta, etc.",
+		features: ["Adjusted gyro filtering", "Adjusted PIDs", "Adjusted rates"],
         applyDefaults: ["INAV_PID_CONFIG", "RC_tuning", "PID_ADVANCED", "FILTER_CONFIG"],
-        settingsMSP: [
-            presets.elementHelper("INAV_PID_CONFIG", "gyroscopeLpf", 3),
-            presets.elementHelper("FILTER_CONFIG", "gyroSoftLpfHz", 30),
-            presets.elementHelper("RC_tuning", "roll_rate", 360),
-            presets.elementHelper("RC_tuning", "pitch_rate", 130),
-            presets.elementHelper("RC_tuning", "dynamic_THR_PID", 30),
-            presets.elementHelper("RC_tuning", "dynamic_THR_breakpoint", 1500)
-        ],
+        settingsMSP: [],
+		type: 'flyingwing',
         settings: [
-            {
-                key: "gyro_hardware_lpf",
```

Comment:
@b14ckyy actually I would leave this kind of settings in place, not remove them

Reply:
@DzikuVx it is still there in all presets. It just moved position as i have rewritten the whole list. that's why it shows as removed.

Reply:
as discussed in Discord. The Preset should allow people to apply the default settings to planes after applying their DIFF. And then tune from there with the new autotune feature. This should go into the Update steps of the release notes.